xbmc/cores/RetroPlayer/shaders/ShaderPresetFactory.cpp

```diff
@@ -140,11 +140,11 @@ void CShaderPresetFactory::UpdateAddons()
     {
       for (const auto& extension : addonPtr->GetExtensions())
         RegisterLoader(addonPtr.get(), extension);
-      m_shaderAddons.emplace(std::move(addonId), std::move(addonPtr));
+      m_shaderAddons.emplace(addonId, std::move(addonPtr));
     }
     else
     {
-      m_failedAddons.try_emplace(std::move(addonId), std::move(addonPtr));
+      m_failedAddons.try_emplace(addonId, std::move(addonPtr));
```

Comment:
A better fix is imo to change line 127 to `std::string addonId =`, not to remove `std::move` here.